.github/workflows/continuous-integration.yml

```diff
@@ -37,6 +37,7 @@ jobs:
   testing:
     name: Testing
     runs-on: ubuntu-latest
+    needs: linting
```

Comment:
```suggestion
```
I think we still want to run `linting`, `testing`, and `build` in parallel. It looks like that's how it is in the [Jenkinsfile](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/Jenkinsfile#L23-L41). This way, the `Build` job also won't need to wait on the other two jobs to finish.